src/character/gear/EditableItemEntry.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useContext, useEffect, useState } from "react";
+import styled, { css } from "styled-components";
+import { Theme } from "../../Theme";
+import { CustomItem, CustomItemData } from "../../utils/types/CustomItem";
+import { Modal, TextArea } from "react-aria-components";
+import { CharacterContext } from "../../DeadlandsCompanion";
+import { Button } from "../../shared/buttons/Button";
+import { useLongPress } from "use-long-press";
+import { deleteCustomitem, updateCustomItem } from "./CustomItemService";
+
+const EditableItemEntryWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  border-radius: 4px;
+  ${({}) => css`
+    background-color: ${Theme.Surface[300]};
+  `}
+`;
+type EditableItemEntryProps = {
+  item: CustomItem;
+};
```

Comment:
I would suggest moving the type definition to be placed right above the component definition instead of between the styled-components.